src/main/scala/nl.knaw.dans.easy.solr4files/Command.scala

```diff
@@ -44,6 +41,15 @@ object Command extends App with DebugEnhancedLogging {
     .doIfFailure { case NonFatal(e) => println(s"FAILED: ${ e.getMessage }") }
 
   private def runCommand(app: EasyUpdateSolr4filesIndexApp): Try[FeedBackMessage] = {
-    Success("")
+    commandLine.subcommand
+      .collect {
+        case update @ commandLine.update => app.update(update.bagStore(), update.bagUuid())
+        case delete @ commandLine.delete => app.delete(delete.bagUuid())
+        case init @ commandLine.init => init.bagStore.toOption
+          .map(app.initSingleStore)
+          .getOrElse(app.initAllStores())
+        case commandLine.runService => ???
```

Comment:
Don't forget this one, there is no checkbox in the PR and no TODO either.

Reply:
If you would check your email more often, you would have seen this comment:

https://github.com/DANS-KNAW/easy-update-solr4files-index/pull/1#issuecomment-328852321